docs/modules/mapstore/pages/configuring-a-generic-maploader.adoc

```diff
@@ -0,0 +1,388 @@
+= Using the Generic MapLoader
+:description: With the xref:working-with-external-data.adoc#options[generic MapLoader], you can configure a map to cache data from an external data store. This topic includes an example of how to configure a map with a generic MapLoader that connects to a MySQL database.
+:page-beta: true
+
+{description}
+
+For a list of all supported databases, see the xref:external-data-stores:external-data-stores.adoc#factories[built-in data store factories].
+
+== Before you Begin
+
+You need a xref:external-data-stores:external-data-stores.adoc[data store connection] that's configured on all cluster members.
```

Comment:
It was renamed to data links, see https://github.com/hazelcast/hz-docs/pull/676

Reply:
Renamed to data connection in line with most recent changes.